example/storybook/stories/Notifications/index.tsx

```diff
@@ -198,7 +141,7 @@ export const NotificationsScreen = () => {
         </TouchableOpacity>
       </View>
 
-      {events.map((event, idx) => (
+      {(events as unknown as Event[]).map((event: Event, idx: number) => (
```

Comment:
this is ugly and fallout from some typing a bit higher in the app tree. will detail it below in this PR

Reply:
Your return type [here](https://github.com/lifeomic/react-native-sdk/pull/238/files#diff-21335a07ddbab7e65d193bfa347f491ddcab07229361b14275778f3e0d92d759R15) says that `events` can be either a single generic DOM `Event` or an array of objects. Why is `Event` an option? Is this defined by the notification library?

Reply:
yes i think it must be supplied by the notifications library. one of the type errors that im getting ultimately bubbles down to: `Type '(events: Event[]) => (Event | { type: string; notification: Notification; })[]' is not assignable to type '(prevState: Event[]) => Event[]'.`

Reply:
yes i think it must be supplied by the notifications library. one of the type errors that im getting ultimately bubbles down to: `Type '(events: Event[]) => (Event | { type: string; notification: Notification; })[]' is not assignable to type '(prevState: Event[]) => Event[]'.`

Reply:
oops i said wrong - the Event type is defined here: https://github.com/lifeomic/react-native-sdk/pull/238/files#diff-1f54bc7376235dcc4820ec4b651589e27ef330fe6f71a5dcf182a3682cdad53cR12